### PR TITLE
Error in color values of "--ring" variable in the theme

### PR DESCRIPTION
**Pull request by zsio**
Error in color values of "--ring" variable in the theme

### CODE REVIEW

File: apps/www/public/registry/colors/index.json [hunk 1]
```diff
@@ -45,9 +45,9 @@
       "scale": 300,
       "hex": "#cbd5e1",
       "rgb": "rgb(203,213,225)",
-      "hsl": "hsl(212.7,26.8%,83.9)",
+      "hsl": "hsl(212.7,26.8%,83.9%)",
       "rgbChannel": "203 213 225",
-      "hslChannel": "hsl(212.7,26.8%,83.9)"
+      "hslChannel": "hsl(212.7,26.8%,83.9%)"
```
Comment (dan5py) [inline apps/www/public/registry/colors/index.json:50]: This should be:
```json
"hslChannel": "212.7 26.8% 83.9%"
```

File: apps/www/public/registry/colors/slate.json [hunk 2]
```diff
@@ -84,9 +84,9 @@
       "destructive-foreground": "210 40% 98%",
       "border": "217.2 32.6% 17.5%",
       "input": "217.2 32.6% 17.5%",
-      "ring": "hsl(212.7,26.8%,83.9)"
+      "ring": "212.7,26.8%,83.9%"
```
Comment (dan5py) [inline apps/www/public/registry/colors/slate.json:87]: This should be:

```json
"ring": "212.7 26.8% 83.9%"
```

**Review by dan5py — CHANGES_REQUESTED** [review]
Thanks @zsio, can you please update this with the changes that I suggested.

File: apps/www/public/registry/colors/index.json (hunk 1 above)
Comment (zsio) [reply apps/www/public/registry/colors/index.json]: okay

File: apps/www/public/registry/colors/slate.json (hunk 2 above)
Comment (zsio) [reply apps/www/public/registry/colors/slate.json:87]: Here has been modified

Comment (dan5py) [reply apps/www/public/registry/colors/slate.json:87]: I mean without commas

File: apps/www/public/registry/colors/index.json [hunk 3]
```diff
@@ -45,9 +45,9 @@
       "scale": 300,
       "hex": "#cbd5e1",
       "rgb": "rgb(203,213,225)",
-      "hsl": "hsl(212.7,26.8%,83.9)",
+      "hsl": "hsl(212.7,26.8%,83.9%)",
       "rgbChannel": "203 213 225",
-      "hslChannel": "hsl(212.7,26.8%,83.9)"
+      "hslChannel": "212.7,26.8%,83.9%"
```
Comment (dan5py) [inline apps/www/public/registry/colors/index.json:50]: Also here without it should be without commas

**Review by dan5py — CHANGES_REQUESTED** [review]

File: apps/www/public/registry/colors/index.json (hunk 3 above)
Comment (zsio) [reply apps/www/public/registry/colors/index.json]: Okay, I have corrected it.

File: apps/www/registry/colors.ts [hunk 4]
```diff
@@ -35,7 +35,7 @@ export const colors = {
       scale: 300,
       hex: "#cbd5e1",
       rgb: "rgb(203,213,225)",
-      hsl: "hsl(212.7,26.8%,83.9)",
+      hsl: "hsl(212.7,26.8%,83.9%)",
```
Comment (shadcn) [inline apps/www/registry/colors.ts:38]: ha! This has been bugging me for a while and I did't even see this. Thanks for the fix.

**Review by shadcn — APPROVED** [review]

**Pull request merged**